roles/common/defaults/main.yml

```diff
@@ -17,6 +17,7 @@ common:
   serial_console:
     enabled: True
     name: ttyS0
+    baud_rate: 9600
```

Comment:
its just a default, so I'm not super concerned, but we should try and default to the most common baud_rate across hardware stacks that we know of.